src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -680,8 +681,20 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
     }
 
     private fun checkLabel(holder: RsAnnotationHolder, label: RsLabel) {
-        if (!hasResolve(label)) return
-        RsDiagnostic.UndeclaredLabelError(label).addToHolder(holder)
+        if (hasResolve(label)) {
+            RsDiagnostic.UndeclaredLabelError(label).addToHolder(holder)
+        }
+        val labelName = label.referenceName
+        if (labelName.isInvalidLabelName(label.edition)) {
+            RsDiagnostic.InvalidLabelName(label, labelName).addToHolder(holder)
+        }
+    }
+
+    private fun checkLabelDecl(holder: RsAnnotationHolder, labelDecl: RsLabelDecl) {
+        val labelName = labelDecl.name
+        if (labelName?.isInvalidLabelName(labelDecl.edition) == true) {
+            RsDiagnostic.InvalidLabelName(labelDecl, labelName).addToHolder(holder)
```

Comment:
```suggestion
            RsDiagnostic.InvalidLabelName(labelDecl.quoteIdentifier, labelName).addToHolder(holder)
```

not to highlight `:` after label name

Reply:
Fixed:
<img width="236" alt="スクリーンショット 2021-11-21 8 48 20" src="https://user-images.githubusercontent.com/1121855/142744274-26e5ea88-f4ea-4254-90be-b588ff7a3ac9.png">